lua/wfp_verify.lua

```diff
@@ -219,27 +219,35 @@ function verifyWFP(tTarget, tWfpControl, iChiptype, atWfpConditions, tPlugin, tF
     end
     -- print final verify result and return
     print_verify_summary(atFlashDataTable, tLog)
-    print("VerifyWfp result: " .. tostring(fVerified))
 
     return fVerified
 end
 
 function print_verify_summary(atFlashDataTable, tLog)
+    local ulSuccessCount = 0
+    local ulFailCount = 0
     tLog.info("Verification summary:")
     tLog.info("===========================================")
     for strChunkKey, atFlashData in pairs(atFlashDataTable) do
         if atFlashData['atChunkList'] ~= nil then
-            tLog.info("Flash Bus: "..atFlashData['tBus'] .." ChipSelect: " .. atFlashData['ulChipSelect'] .." Unit: "..atFlashData['ulUnit'])
+            tLog.info("Flash Bus: "..atFlashData['tBus'] .." Unit: "..atFlashData['ulUnit'].." ChipSelect: " .. atFlashData['ulChipSelect'])
             tLog.info("-------------------------------------------")
         end
         for _, tChunk in ipairs(atFlashData['atChunkList']) do
             if tChunk.verified == true then
                 tLog.info("    %s Chunk 0x%08x - 0x%08x: OK ", tChunk.strType, tChunk.ulOffset, tChunk.ulEndOffset)
+                ulSuccessCount = ulSuccessCount +1
             else
                 tLog.info("    %s Chunk 0x%08x - 0x%08x: FAILED ", tChunk.strType, tChunk.ulOffset, tChunk.ulEndOffset)
+                ulFailCount = ulFailCount +1
             end
         end
+        print("")
     end
+    tLog.info("===========================================")
+    tLog.info("    Passes:   %s", ulSuccessCount)
```

Comment:
"Passes" does not mean what we want here. 
How about "Passed" and "Failed"? 
Or how about
Files/Blocks successfully verified: 
Files/Blocks failed to verify: